### ISSUE TITLE
Remove Enter Shortcut to run laps

### ISSUE THREAD

**Issue by JonBeagles** [christopherh0rn/f1-revisited-bugs#61]
So apparently if you press Enter on the Qualifying screen it runs the lap - I'm not sure if this is a website shortcut, or that's something the user has got set up themselves...

...but if it is a shortcut on the website can we make it 'clickable only', so the player has to click for the lap to run and can't just press Enter.

**Comment by christopherh0rn** [comment]
This is actually a web standards thing, if one is entering values into a form (so one of the inputs has focus) and one presses enter it will submit the form. See [here](https://www.w3.org/TR/2018/SPSD-html5-20180327/forms.html#implicit-submission).

I can alter this behaviour but it would be a non-standards compliant.

**Comment by christopherh0rn** [comment]
An explainer: https://www.smashingmagazine.com/2019/01/web-standards-guide/

TLDR: accessibility, interoperability, simplicity, and lots of good things.

If you want we can overrule but it does feel wrong particularly as it's not something I built, it's the default behaviour built into the browser.

**Comment by JonBeagles** [comment]
Ok cool - let's just keep it as it is.

Is there any way for players to disable that in their own browsers?
________________________________
